lambda/snapshots_tool_utils.py

```diff
@@ -362,3 +364,98 @@ def copy_remote(snapshot_identifier, snapshot_object):
             CopyTags = True)
 
     return response
+
+
+def get_all_automated_snapshots(client):
+    global _AUTOMATED_BACKUP_LIST
+    if len(_AUTOMATED_BACKUP_LIST) == 0:
+        response = paginate_api_call(
+            client, 'describe_db_snapshots', 'DBSnapshots', SnapshotType='automated'
+        )
+        _AUTOMATED_BACKUP_LIST = response['DBSnapshots']
+
+    return _AUTOMATED_BACKUP_LIST
+
+
+def avail_db_snapshot(
```

Comment:
```suggestion
def available_db_snapshot(
```
> 

If a function argument's name clashes with a reserved keyword, it is generally better to append a single trailing underscore rather than use an abbreviation or spelling corruption. 
https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments